web/pages/index.tsx

```diff
@@ -1,35 +1,195 @@
-import { FacebookAuthProvider, signInWithPopup } from "firebase/auth";
-import Head from "next/head";
+import {
+  FacebookAuthProvider,
+  GoogleAuthProvider,
+  signInWithPopup,
+  signOut,
+  User,
+} from "firebase/auth";
+import {
+  Button,
+  Flex,
+  Stack,
+  Text,
+  Divider,
+  Popover,
+  PopoverTrigger,
+  PopoverContent,
+  Link,
+} from "@chakra-ui/react";
 import { auth } from "../utils/firebase/firebaseClient";
-import { trpc } from "../utils/trpc";
-import { useAuth } from "../context/auth";
+import React from "react";
+//import { trpc } from "../utils/trpc";
+//import { useAuth } from "../context/auth";
 
 export default function Home() {
-  async function handleLogin() {
+  const [user, setUser] = React.useState<User | null>(null);
+
+  async function handleLoginFacebook() {
     const provider = new FacebookAuthProvider();
     const result = await signInWithPopup(auth, provider);
-    const user = result.user;
+    setUser(result.user);
```

Comment:
The `useAuth` hook already implemented will handle all the logic for setting to null, etc. You won't need to set the object manually.